Epsilon/Component/KpiMatrixComponentFetcher.cs

```diff
@@ -0,0 +1,143 @@
+﻿using Epsilon.Abstractions.Component;
+using Epsilon.Canvas.Abstractions.Model;
+using Epsilon.Canvas.Abstractions.Model.GraphQl;
+using Epsilon.Canvas.Abstractions.Service;
+using Microsoft.Extensions.Configuration;
+
+namespace Epsilon.Component;
+
+public class KpiMatrixComponentFetcher : ComponentFetcher<KpiMatrixCollection>
+{
+    private const string GetUserKpiMatrixOutcomes = @"
+        query GetUserKpiMatrixOutcomes {
+          course(id: $courseId) {
+            submissionsConnection {
+              nodes {
+                assignment {
+                  name
+                  modules {
+                    name
+                  }
+                }
+                rubricAssessmentsConnection {
+                  nodes {
+                    assessmentRatings {
+                      points
+                      outcome {
+                        _id
+                        title
+                        masteryPoints
+                      }
+                    }
+                  }
+                }
+              }
+            }
+          }
```

Comment:
You should use allCourses and filter on studentId instead of targeting a single course.